roboteam_ai/src/skills/ShootFreeKick.cpp

```diff
@@ -75,7 +75,8 @@ Skill::Status ShootFreeKick::onUpdate() {
         case SHOOTING: {
             if (! isShot()) {
                 Vector2 target = rtt::ai::world::field->getPenaltyPoint(false);
-                robot->getShotController()->makeCommand(robot->getShotController()->getShotData(* robot, target, true), command);
+                auto shotData = robot->getShotController()->getShotData(* robot, target, true);
+                command = shotData.makeROSCommand();
```

Comment:
So if I understand correctly you're removing the call to the shotcontroller object that makes the command, so we're not keeping track correctly of the right variables for accelerationLimiter/VelocityLimiter anymore, as that is not done in the shotcontroller itself.